### PR TITLE
Adds test coverage to CharacterUtils

### PR DESCRIPTION
**Pull request by jas-valgotar**
-Added class "CharacterUtilsTests.cs" to improve code coverage for class "CharacterUtils"

### CODE REVIEW

**Comment by lesaltzm** [comment]
Can you write a more meaningful title/description for this PR?

File: src/tests/Microsoft.PowerFx.Core.Tests/CharacterUtilsTests.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        public void MakeSafeForFormatStringTests(string expected, string input)
+        {
+            var actual = CharacterUtils.MakeSafeForFormatString(input);
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void HasSpacesTest()
+        {
+            var actual = CharacterUtils.HasSpaces(" ");
```
Comment (lesaltzm) [inline src/tests/Microsoft.PowerFx.Core.Tests/CharacterUtilsTests.cs:64]: What about other weird spaces (nbsp, newlines, etc..)

**Comment by BruceHaley** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Connectors.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Core.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Interpreter.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.LanguageServerProtocol.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Transport.Attributes.dll**

File: src/tests/Microsoft.PowerFx.Core.Tests/CharacterUtilsTests.cs (hunk 1 above)
Comment (jas-valgotar) [reply src/tests/Microsoft.PowerFx.Core.Tests/CharacterUtilsTests.cs]: Added new test cases for the same!

**Comment by BruceHaley** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Connectors.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Core.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Interpreter.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.LanguageServerProtocol.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Transport.Attributes.dll**

File: src/tests/Microsoft.PowerFx.Core.Tests/CharacterUtilsTests.cs (hunk 1 above)
Comment (jas-valgotar) [reply src/tests/Microsoft.PowerFx.Core.Tests/CharacterUtilsTests.cs]: Added new test cases for nbsp and tab!

**Review by MikeStall — APPROVED** [review]
:shipit:

**Comment by lesaltzm** [comment]
> Can you write a more meaningful title/description for this PR?

Maybe change the title to something like "Add test coverage to CharacterUtils"?
That's more meaningful than your name + the branch name, generally, and will help when folks go back through and try to find old PRs.

**Review by lesaltzm — APPROVED** [review]

**Comment by BruceHaley** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Connectors.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Core.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Interpreter.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.LanguageServerProtocol.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Transport.Attributes.dll**

**Pull request merged**